app/design/adminhtml/Magento/backend/web/css/source/forms/fields/_control-table.less

```diff
@@ -122,6 +122,12 @@
         }
     }
 
+    td {
+        .admin__field-control  {
```

Comment:
Why this class should be added?

Reply:
Hi @osrecio 

I have changed only one file : https://github.com/magento/magento2/pull/17852/commits/a34ab10900d05f3ef22add51185f830f65037428

Reply:
Can you update your branch with last changes because I'm viewing 2 files modified in this PR : https://github.com/magento/magento2/pull/17852/files

Reply:
Means do you want I will remove that change from my branch?

Reply:
Update your branch 2.2-develop with lasts changes and rebase with your branch and push.

Reply:
Can you please share with me steps. How can I rebase

Reply:
Check this video: https://www.youtube.com/watch?v=mJDCL0uzIpY&list=PLDvMskiz68Q3lLDOe2pMXssh8iVTinRN7&index=1

Reply:
I am getting below  error when push branch

git push
key_load_public: invalid format
Counting objects: 30, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (24/24), done.
Writing objects: 100% (30/30), 3.22 KiB | 3.22 MiB/s, done.
Total 30 (delta 17), reused 7 (delta 2)
remote: Resolving deltas: 100% (17/17), completed with 16 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/2.2-develop.
remote: error: You're not authorized to push to this branch. Visit https://help.github.com/articles/about-protected-branches/ for more information.
To github.com:magento/magento2.git
 ! [remote rejected]         2.2-develop -> 2.2-develop (protected branch hook declined)
error: failed to push some refs to 'git@github.com:magento/magento2.git'

Reply:
I sent you a ping via EngCom Slack. We can continue by this way.